cardano-api/src/Cardano/Api/TxBody.hs

```diff
@@ -704,7 +709,14 @@ data TxBody era where
      ShelleyTxBody
        :: ShelleyBasedEra era
        -> Ledger.TxBody (ShelleyLedgerEra era)
-       -> Maybe Shelley.MetaData
+
+          -- The 'Ledger.Metadata' is really the /auxiliary/ data, it consists
```

Comment:
We _could_ use haddock here:

```suggestion
          -- | The 'Ledger.Metadata' is really the /auxiliary/ data, it consists
```

Reply:
Haddock comments do not fit here. And this is a comment on the internal representation.